golang/go-guestbook/kubernetes-manifests/guestbook-mongodb.deployment.yaml

```diff
@@ -6,7 +6,7 @@
 kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: guestbook-mongodb
+  name: go=guestbook-mongodb
```

Comment:
I think this should be `go-` instead of `go=`?

Reply:
Thanks, for the catch!